server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/GetMessageListMethodTest.java

```diff
@@ -81,6 +82,63 @@ public void setup() throws Throwable {
     public void teardown() {
         jmapServer.stop();
     }
+
+    @Test
+    public void getMessagesShouldBehaveAsIfMailboxDidNotExistWhenUsedOnAnOtherUserMailbox() throws Exception {
```

Comment:
I guess, at this point, the test fails. So you have to put a @Ignore on it until next commit.